clientlib-generator.js

```diff
@@ -0,0 +1,41 @@
+var clientlib = require("aem-clientlib-generator");
```

Comment:
Why are we using the 'aem-clientlib-generator' library?
@egmox already working on the automated theme. 
The Canvas theme has been published on NPMJS (https://www.npmjs.com/package/@aemforms/af-canvas-theme), and it can be downloaded from there.

Reply:
This library is used to generate an AEM clientlib from CSS /JS files. The published theme is not an AEM clientlib